lisa/tools/free.py

```diff
@@ -96,3 +96,6 @@ def get_total_memory(self) -> str:
         group = find_group_in_lines(output, self._mem_pattern)
         total_memory = group["total"]
         return total_memory
+
+    def log_memory_stats_mb(self) -> None:
```

Comment:
return the value as float type, so the logged value can be used in code.

Reply:
It actually logs a bunch of values. For example:

```
              total        used        free      shared  buff/cache   available
Mem:          55765        2904       23381           1       29479       52212
Swap:             0           0           0
```